realsense2_camera/src/realsense_node_factory.cpp

```diff
@@ -204,9 +220,45 @@ void RealSenseNodeFactory::onInit()
     }
 }
 
+void RealSenseNodeFactory::onInit()
+{
+    auto privateNh = getPrivateNodeHandle();
+    auto nodeHandle = getNodeHandle();
+    bool wait_for_usb_resetter = true;
+    privateNh.param("usb_port_id", _usb_port_id, std::string(""));
+    privateNh.param("wait_for_usb_resetter", wait_for_usb_resetter, true);
+
+    if (wait_for_usb_resetter)
+    {
+        ROS_INFO_STREAM(__FILE__ << " " << __LINE__ << "realsense_camera: device " << _usb_port_id
+                                 << " wait_for_usb_resetter enabled!");
+        ros::SubscriberStatusCallback connect_cb = boost::bind(&RealSenseNodeFactory::connectCb, this);
```

Comment:
I think this may be better as a service call than do the pub/sub for a synchronous call.